### PR TITLE
Newsletters: add 221 (2022-10-12)

### PR DESCRIPTION
**Pull request by harding**
- [x] Add lede, update releases/RCs, add topic links @harding 
- [x] Bitcoin Core PR Review Club @LarryRuane 
- [x] LND 6500 @adamjonas

### CODE REVIEW

**Comment by LarryRuane** [comment]
https://github.com/harding/bitcoinops.github.io/compare/2022-10-12-newsletter...LarryRuane:bitcoinops.github.io:2022-10-12-review-club?expand=1

File: _posts/en/newsletters/2022-10-12-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        would Alice be able to spend the money.
+
+        This would ensure Bob was able to claim his funds at any time up
+        until the trigger transaction had received the agreed-upon
+        number of confirmations, even if months had passed
+        since a normal HTLC would've timed out.  If Bob is adequately
+        compensated for his waiting, then it's ok if Alice remains
+        offline all that time.  For an HTLC routed from Alice though Bob
+        onto some distant node, only the channel between Alice and Bob
+        would be affected---every other channeL would settle the HTLC
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-10-12-newsletter.md:46]:
```suggestion
        would be affected---every other channel would settle the HTLC
```

File: _posts/en/newsletters/2022-10-12-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        this allows Alice to publish the latest state secure in the
+        knowledge that Bob can't publish a contradictory version,
+        allowing her to safely go offline after her publication.
+
+    Law's proposals were still receiving initial feedback as this
+    description was being written.
+
+- **Recommendations for unique address servers:** Ruben Somsen
+  [posted][somsen post] to the Bitcoin-Dev mailing list a
+  [document][somsen gist] another suggestion for how users can avoid
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-10-12-newsletter.md:77]:
```suggestion
  [document][somsen gist] for another suggestion for how users can avoid
```

File: _posts/en/newsletters/2022-10-12-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  [output linking][topic output linking] without trusting a third-party
+  service or using an cryptographic protocol that's not currently widely
+  supported, like [BIP47][] or [silent payments][topic silent payments].
+  The recommend method is especially intended for wallets that are
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-10-12-newsletter.md:81]:
```suggestion
  The recommended method is especially intended for wallets that are
```

**Review by LarryRuane — COMMENTED** [review]
I still can't push to this branch, but I addressed @bitschmidty's review comments in this commit, please cherry-pick -- this is a force-push (replacement) of the [existing commit](cee72dd3ec59e0c70fd95be78ce8a7ebed398220))
https://github.com/LarryRuane/bitcoinops.github.io/commit/b59674c040dfa806ddc8bbc2e3f52677ee7af820
(sorry, I probably should have made that a separate commit)

File: _posts/en/newsletters/2022-10-12-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+commonly occurs when the node is configured to not use IPv4 or IPv6
+(for example, `-onlynet=tor`)."
+  a0link="https://bitcoincore.reviews/26114#l-27"
+
+  q1="What observable behavior change does this PR introduce?
+What kinds of addresses do we add to `AddrMan`, and under what circumstances?"
+  a1="The node, rather than immediately adding the fixed seeds to its
+`AddrMan` and making full connections to some of them, instead
+makes `AddrFetch` connections to a some of them,
+and adds the _returned addresses_ to `AddrMan`. (`AddrFetch` are
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2022-10-12-newsletter.md:127]: I think this is better as a parenthetical.

File: _posts/en/newsletters/2022-10-12-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+operators would be less likely to have multiple simultaneous IBD peers,
+which reduces the load (resource requirements) on their nodes."
+  a2link="https://bitcoincore.reviews/26114#l-77"
+
+  q3="The DNS seed nodess are expected to be responsive and serve
+up-to-date addresses of Bitcoin nodes. Why doesn’t this help
+a -onlynet=tor node?"
+  a3="The DNS seed nodes provide only IPv4 and IPv6 addresses;
+they're not able to supply any other type of address."
+  a3link="https://bitcoincore.reviews/26114#l-89"
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2022-10-12-newsletter.md:154]: @bitschmidty's suggestion for line 35 seems better to me, so I'm going to make that change, but I think this is ok to revert even after merge if @LarryRuane prefers the original line 89.

**Comment by harding** [comment]
Made edits (thanks @bitschmidty @adamjonas @LarryRuane ), added the LND emergency release, added topic links, and added lede.  Thanks everyone!

**Comment by bitschmidty** [comment]
Looks good, thank you @harding @adamjonas @LarryRuane !

**Pull request merged**